src/wlapplication.cc

```diff
@@ -1293,14 +1221,28 @@ void WLApplication::handle_commandline_parameters() {
 			exit(1);
 		}
 	}
+
+	if (commandline_.count("language")) {
+		const std::string& lang = commandline_["language"];
+		if (!lang.empty()) {
+			set_config_string("language", lang);
+		} else {
+			init_language();
+			throw ParameterError(
+			   CmdLineVerbosity::None,
+			   (boost::format(_("Empty value of command line parameter: %s")) % "language").str());
```

Comment:
Having such strings in multiple places is redundant if we change it one day. Transifex has fuzzy matches though so you can just replace the placeholder with the option name in every occurrence. Alternatively pull out some inline functions.